### PR TITLE
fix errors reported by @jammyjamjamman

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: book/Edition02/stories/enchanted_balls.tex [hunk 1]
```diff
@@ -53,13 +53,13 @@ \chapter{The Search for the Enchanted Balls}
...
 
 \timesep
 
-After two days, Katharine and Jacob were starving and thirsty. They agreed to start searching for a food and water supply. Jacob opened the lid far enough to get out. After being loosened by the belt buckle, the lid separated easily from the top of the container. They both crawled out and then stretched their bodies and tails as much as possible. They had never been trapped for such a  long time.
+After two days, Katharine and Jacob were starving and thirsty. They agreed to start searching for a food and water supply. Already loosened, Jacob easily pushed up on the container lid. They both crawled out and then stretched their bodies and tails as much as possible. They had never been trapped for such a  long time.
```
Comment (Jammyjamjamman) [inline book/Edition02/stories/enchanted_balls.tex:62]: All these edits look really good

File: book/Edition02/stories/the_prisoner.tex [hunk 2]
```diff
@@ -6,7 +6,7 @@ \chapter{The Prisoner}
 
 One morning, a long time ago, one such prisoner was eating breakfast. A speck of light suddenly appeared several inches in front of zem. It momentarily hurt zirs eyes but the pain quickly subsided. Several seconds later the light changed from a speck to the same size as one of the blocks in the wall.
 
-Ze was so very bored much of the time, so although frightened, ze continued watching the light. The stone behind it slid slowly out of the wall and fell to the ground. It began to grow small arms, legs, a head; and finally, a mouth.
+Ze was frightened by the light, but because ze was normally so very bored during his imprisonment, ze welcomed his fear, deciding it was an exciting change. Ze continued watching the light. The stone behind it slid slowly out of the wall and fell to the ground. It began to grow small arms, legs, a head; and finally, a mouth.
```
Comment (Jammyjamjamman) [inline book/Edition02/stories/the_prisoner.tex:9]: I like this change :)

**Review by Jammyjamjamman — APPROVED** [review]
All looks good to me

**Comment by andy5995** [comment]
> All looks good to me

Thanks for reviewing! I'll let it ferment for a day or two and probably won't make any more changes.

**Pull request merged**